cmake/emu/simics.cmake

```
@@ -0,0 +1,30 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+find_program(
+  SIMICS
+  NAMES simics
+  )
+
+set(SIMICS_SCRIPT_PATH $ENV{SIMICS_SCRIPT_PATH})
```

perhaps use `zephyr_get()`, that will allow users to set the path either through environment, as well as using `-DSIMICS_SCRIPT_PATH=<path>`.

```suggestion
zephyr_get(SIMICS_SCRIPT_PATH SYSBUILD GLOBAL)
```

---

done